helm/chaos-mesh/templates/webhook-configuration.yaml

```diff
@@ -1,3 +1,27 @@
+{{- $altNames := list ( printf "%s.%s.svc" (include "chaos-mesh.svc" .) .Release.Namespace )  }}
+{{- $ca := genCA "chaos-mesh-ca" 3650 }}
+{{- $cert := genSignedCert "chaos-mesh-ca" nil $altNames 3650 $ca }}
```

Comment:
10 year？
Maybe we should also change the `duration` of `Certificate`

https://github.com/pingcap/chaos-mesh/blob/ad4f1204a410422e0a93f3aba6b143f42cee15df/helm/chaos-mesh/templates/webhook-configuration.yaml#L108

Reply:
got